server/planning/commands/purge_expired_locks_test.py

```diff
@@ -24,102 +26,138 @@
 
 # TODO: Add Assignments
 class PurgeExpiredLocksTest(TestCase):
+    app_config = {
```

Comment:
If we're not adding/modifying the `app_config`, then there's no need to re-define it here as it's already defined in the parent class